app/assets/javascripts/pure_admin/partial_refresh.js

```diff
@@ -4,7 +4,8 @@ PureAdmin.partial_refresh = {
   loadingTimer: {},
 
   init: function() {
-    var matchElem = '.js-partial-refresh';
+    var matchElem = '.pagination, .js-partial-refresh';
```

Comment:
If pagination needs to trigger a partial refresh we should be adding the `.js-partial-refresh` class to the pagination element rather than to the `.pagination` class to the JS. `.js-` classes are used for creating js integrations without affecting styling.